libbeat/processors/actions/add_fields.go

```diff
@@ -71,18 +71,20 @@ func NewAddFields(fields common.MapStr, shared bool, overwrite bool) processors.
 	return &addFields{fields: fields, shared: shared, overwrite: overwrite}
 }
 
-func (af *addFields) Run(event *beat.Event) (*beat.Event, error) {
+func (af *addFields) Run(event *beat.Event) (_ *beat.Event, err error) {
```

Comment:
Why use named parameters? We tend to avoid them in the codebase.

Reply:
that's a leftover from my first approach with the whole backup of the event. I will remove this.